Chapter02/Activity2.02/app/src/main/java/com/example/loginactivity/MainActivity.kt

```diff
@@ -43,15 +54,18 @@ class MainActivity : AppCompatActivity() {
                     welcomeIntent.putExtra(PASSWORD_KEY, passWord)
 
                     //Reset text fields to blank
-                    user_name.text.clear()
-                    password.text.clear()
+                    this.userName.text.clear()
```

Comment:
@drulabs - Could you make a small note in the chapter explaining the field/variable access you are using here with 'this' for these 2 fields/variables as opposed to the local variables as for a new person this might be confusing. Thank you

Reply:
Actually you don't have to update the chapter if you just change the name of the local variable to resolve the naming issue.

Reply:
Well, obviously you will have to update the code example in the chapter, but you won't have to explain what 'this' is - thanks